tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/settings.controller.js

```diff
@@ -1265,7 +1284,67 @@
                     getTaxonomyHierarchy(data);
                 }
             }
-
+            vm.extraMatterFields = [];
+            function getAdditionalMatterProperties(data) {
+                vm.lazyloader = false;
```

Comment:
Get the matter extra field properties with metadata to set the settings for fields data like isDisplayInUi and IsMandatory to true and false value.